front50-s3/src/main/java/com/netflix/spinnaker/front50/config/S3Config.java

```diff
@@ -138,6 +140,11 @@ public PipelineDAO pipelineDAO(StorageService storageService, Registry registry)
   }
 
   @Bean
+  public PipelineTemplateDAO pipelineTemplateDAO(StorageService storageService, Registry registry) {
+    return new DefaultPipelineTemplateDAO(storageService, Schedulers.from(Executors.newFixedThreadPool(25)), 10000, registry);
```

Comment:
Technically you've got a different refresh interval between S3 and GCS ... my PR will clean this up as well but I'd suggest dropping it down to 60s.

Any call to `all()` will do an on-demand refresh of any deltas anyways so it's more about having a reasonably fresh cache in the event of a hystrix fallback.